### PR TITLE
JBPM-8733  Manage section filter tag shows raw query versus named fi…

### PR DESCRIPTION
**Pull request by bxf12315**
…lter

@cristianonicolai @nmirasch please review it.

### CODE REVIEW

File: kie-soup-dataset/kie-soup-dataset-api/src/main/java/org/dashbuilder/dataset/json/DataSetLookupJSONMarshaller.java [hunk 1]
```diff
@@ -93,6 +94,7 @@
         _keysAliasMap.put(FUNCTION_ARGS, Arrays.asList(FUNCTION_ARGS, "terms"));
         _keysAliasMap.put(COLUMN, Arrays.asList(COLUMN, "columnId"));
         _keysAliasMap.put(SOURCE, Arrays.asList(SOURCE, "sourceId"));
+        _keysAliasMap.put(FUNCTION_LABLEVAlUE, Arrays.asList(FUNCTION_LABLEVAlUE, "lableValue"));
```
Comment (cristianonicolai) [inline kie-soup-dataset/kie-soup-dataset-api/src/main/java/org/dashbuilder/dataset/json/DataSetLookupJSONMarshaller.java]: typo in labelValue

File: kie-soup-dataset/kie-soup-dataset-api/src/main/java/org/dashbuilder/dataset/json/DataSetLookupJSONMarshaller.java [hunk 2]
```diff
@@ -62,6 +62,7 @@
 
     private static final String FUNCTION_TYPE = "function";
     private static final String FUNCTION_ARGS = "args";
+    private static final String FUNCTION_LABLEVAlUE = "lableValue";
```
Comment (cristianonicolai) [inline kie-soup-dataset/kie-soup-dataset-api/src/main/java/org/dashbuilder/dataset/json/DataSetLookupJSONMarshaller.java]: perhaps FUNCTION_LABEL_VALUE?

Comment (bxf12315) [reply kie-soup-dataset/kie-soup-dataset-api/src/main/java/org/dashbuilder/dataset/json/DataSetLookupJSONMarshaller.java]: @cristianonicolai Updated, Please review agian.

Comment (cristianonicolai) [reply kie-soup-dataset/kie-soup-dataset-api/src/main/java/org/dashbuilder/dataset/json/DataSetLookupJSONMarshaller.java]: @bxf12315, could you look into adding some tests, please

File: kie-soup-dataset/kie-soup-dataset-api/src/main/java/org/dashbuilder/dataset/json/DataSetLookupJSONMarshaller.java [hunk 3]
```diff
@@ -62,6 +62,7 @@
 
     private static final String FUNCTION_TYPE = "function";
     private static final String FUNCTION_ARGS = "args";
+    private static final String FUNCTION_LABEL_VAlUE = "labelValue";
```
Comment (jstastny-cz) [inline kie-soup-dataset/kie-soup-dataset-api/src/main/java/org/dashbuilder/dataset/json/DataSetLookupJSONMarshaller.java]: Please transform the name to all caps.

File: kie-soup-dataset/kie-soup-dataset-api/src/main/java/org/dashbuilder/dataset/json/DataSetLookupJSONMarshaller.java [hunk 4]
```diff
@@ -399,10 +402,12 @@ public ColumnFilter parseColumnFilter(JsonObject columnFilterJson) {
         }
 
         if (isCoreFilter(functionType)) {
+            String lableValue = columnFilterJson.getString(keySet(FUNCTION_LABEL_VAlUE));
```
Comment (jstastny-cz) [inline kie-soup-dataset/kie-soup-dataset-api/src/main/java/org/dashbuilder/dataset/json/DataSetLookupJSONMarshaller.java]: lableValue -> labelValue

**Comment by bxf12315** [comment]
@cristianonicolai @jstastny-cz Updated, Please review it.

File: kie-soup-dataset/kie-soup-dataset-api/src/main/java/org/dashbuilder/dataset/json/DataSetLookupJSONMarshaller.java [hunk 5]
```diff
@@ -49,50 +49,17 @@
 import org.dashbuilder.json.JsonObject;
 import org.dashbuilder.json.JsonType;
 import org.dashbuilder.json.JsonValue;
+import static org.dashbuilder.dataset.json.DataSetLookupJSONMarshallerConstants.*;
 
 public class DataSetLookupJSONMarshaller {
 
-    private static final String UUID = "dataSetUuid";
```
Comment (cristianonicolai) [inline kie-soup-dataset/kie-soup-dataset-api/src/main/java/org/dashbuilder/dataset/json/DataSetLookupJSONMarshaller.java]: @bxf12315 I don't see much point in this refactor as these constants are private anyway.

File: kie-soup-dataset/kie-soup-dataset-api/src/main/java/org/dashbuilder/dataset/filter/CoreFunctionFilter.java [hunk 6]
```diff
@@ -24,6 +24,7 @@
 public class CoreFunctionFilter extends ColumnFilter {
 
     protected CoreFunctionType type = null;
+    protected String labelValue = "";
```
Comment (cristianonicolai) [inline kie-soup-dataset/kie-soup-dataset-api/src/main/java/org/dashbuilder/dataset/filter/CoreFunctionFilter.java]:
```suggestion
    protected String labelValue = null;
```

File: kie-soup-dataset/kie-soup-dataset-api/src/main/java/org/dashbuilder/dataset/json/DataSetLookupJSONMarshaller.java (hunk 5 above)
Comment (bxf12315) [reply kie-soup-dataset/kie-soup-dataset-api/src/main/java/org/dashbuilder/dataset/json/DataSetLookupJSONMarshaller.java]: Unit test need some the field, so I refactor it.

File: kie-soup-dataset/kie-soup-dataset-api/src/main/java/org/dashbuilder/dataset/filter/CoreFunctionFilter.java (hunk 6 above)
Comment (bxf12315) [reply kie-soup-dataset/kie-soup-dataset-api/src/main/java/org/dashbuilder/dataset/filter/CoreFunctionFilter.java]: done

File: kie-soup-dataset/kie-soup-dataset-api/src/main/java/org/dashbuilder/dataset/json/DataSetLookupJSONMarshaller.java (hunk 5 above)
Comment (cristianonicolai) [reply kie-soup-dataset/kie-soup-dataset-api/src/main/java/org/dashbuilder/dataset/json/DataSetLookupJSONMarshaller.java]: you could just make the needed constants protected?

**Comment by bxf12315** [comment]
@cristianonicolai updated, Please review it.

**Comment by cristianonicolai** [comment]
Jenkins execute full downstream build

**Comment by cristianonicolai** [comment]
@bxf12315 can you please rebase and update the JIRA reference to a JBPM one, thanks.

**Pull request merged**